website/docs/plugin/framework/migrating/resources/state-upgrade.mdx

```diff
@@ -0,0 +1,206 @@
+---
+page_title: 'Resources - State Upgrading: Migrating from SDKv2 to the Framework'
+description: >-
+  State upgraders let users update resources provisioned with old schema configurations.
+  Migrate resource StateUpgraders in SDKv2 to UpgradeState in the plugin Framework.
+---
+
+# State Upgraders
+
+When you update a resource's implementation in your provider, some changes may not be compatible with old versions. You
+can create state upgraders to help users migrate resources provisioned with old schema configurations. Refer to
+[State Upgrade](/plugin/framework/resources/state-upgrade) in the Framework documentation for details.
```

Comment:
Do state upgraders "help users migrate resources" or do they do it automatically?

Reply:
Good point. Have reworded as:

> You can create state upgraders to automatically migrate resources provisioned with old schema configurations.